### PR TITLE
update default server to use production API service

### PR DESCRIPTION
**Pull request by Ulexus**
Use `api.turnkey.com` instead of the old beta service.

### CODE REVIEW

**Review by r-n-o — APPROVED** [review]
🙌

**Review by andrewkmin — APPROVED** [review]
🙌

**Comment by Ulexus** [comment]
D'oh.  That file is autogenerated...

File: pkg/api/client/activities/public_api_service_get_activity_responses.go [hunk 1]
```diff
@@ -29,18 +29,6 @@ func (o *PublicAPIServiceGetActivityReader) ReadResponse(response runtime.Client
 			return nil, err
 		}
 		return result, nil
-	case 403:
```
Comment (andrewkmin) [inline pkg/api/client/activities/public_api_service_get_activity_responses.go:32]: do you happen to know the context of why some of these response codes were removed here? https://github.com/tkhq/mono/commit/e5a4673032ff26077c6428651085265d0a3159f4

Comment (Ulexus) [reply pkg/api/client/activities/public_api_service_get_activity_responses.go:32]: No clue from me... @timurnkey ?

Comment (r-n-o) [reply pkg/api/client/activities/public_api_service_get_activity_responses.go:32]: I think this was removed to simplify our docs. I don't think this is very useful to single out 403/404s here so I'm fine with letting these go?

**Review by r-n-o — APPROVED** [review]

**Pull request merged**